Makefile.isp

```diff
@@ -19,14 +19,23 @@ install: all
 	install -d $(ISP_PREFIX)/lib
 	install -d $(ISP_PREFIX)/bin
 	install -d $(ISP_PREFIX)/include
+	install -p build/dump_tags $(ISP_PREFIX)/bin/
 	install -p build/md_code $(ISP_PREFIX)/bin/
 	install -p build/md_range $(ISP_PREFIX)/bin/
 	install -p build/md_entity $(ISP_PREFIX)/bin/
 	install -p build/md_asm_ann $(ISP_PREFIX)/bin/
 	install -p build/md_header $(ISP_PREFIX)/bin/
 	install -p build/md_index $(ISP_PREFIX)/bin/
+	install -p build/dump_tags64 $(ISP_PREFIX)/bin/
```

Comment:
This seems difficult to fix because of the current policy-engine code structure, but we should open an issue to make 64bit a runtime flag

Reply:
https://github.com/draperlaboratory/hope-policy-engine/issues/61